tests/test-cases/Group1-Docker-Commands/1-02-Docker-Pull.robot

```diff
@@ -108,7 +108,7 @@ Re-pull a previously rmi'd image
     ${id}=  Get From List  ${words}  2
     ${size}=  Get From List  ${words}  -2
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} rmi ubuntu
```

Comment:
I think this should be `${ubuntu}` since `Pull ubuntu` pulls from harbor. Ditto for the `docker %{VCH-PARAMS} images |grep ubuntu` above and below.